services/gitlab/gitlab-merge-requests.spec.js

```diff
@@ -20,17 +22,63 @@ describe('GitlabMergeRequests', function () {
     }).expect({
       label: 'discussion,enhancement merge requests',
       message: '15 open',
-      color: 'brightgreen',
+    })
+    given({
+      variant: 'open',
+      raw: '-raw',
+      labels: 'discussion,enhancement',
+      mergeRequestCount: 15,
+    }).expect({
+      label: 'open discussion,enhancement merge requests',
+      message: '15',
     })
     given({ variant: 'open', mergeRequestCount: 0 }).expect({
       label: 'merge requests',
       message: '0 open',
-      color: 'blue',
     })
     given({ variant: 'open', mergeRequestCount: 10001 }).expect({
       label: 'merge requests',
       message: 'more than 10k open',
-      color: 'brightgreen',
+    })
+  })
+  describe('auth', function () {
+    cleanUpNockAfterEach()
+
+    const fakeToken = 'abc123'
+    const config = {
+      public: {
+        services: {
+          gitlab: {
+            authorizedOrigins: ['https://gitlab.com'],
+          },
+        },
+      },
+      private: {
+        gitlab_token: fakeToken,
+      },
+    }
+
+    it('sends the auth information as configured', async function () {
+      const scope = nock('https://gitlab.com/')
+        .get('/api/v4/projects/foo%2Fbar/repository/tags?order_by=updated')
+        // This ensures that the expected credentials are actually being sent with the HTTP request.
+        // Without this the request wouldn't match and the test would fail.
+        .matchHeader('Authorization', `Bearer ${fakeToken}`)
+        .reply(200, [{ name: '1.9' }])
+
+      expect(
+        await GitLabTag.invoke(
+          defaultContext,
+          config,
+          { project: 'foo/bar' },
+          {}
+        )
```

Comment:
We should be using the merge requests class here:

```suggestion
        await GitlabMergeRequests.invoke(
          defaultContext,
          config,
          { project: 'foo/bar' },
          {}
        )
```